tools/tertiary-analysis/scmap/scmap_get_std_output.xml

```diff
@@ -0,0 +1,33 @@
+<tool id="scmap_get_std_output" name="scmap get standard output" version="@TOOL_VERSION@+galaxy0"  profile="@PROFILE@">
+    <description>Create final output in standard format to allow for downstream analysis of predicted labels</description>
```

Comment:
expand to "analysis of predicted labels by tools of the EBI gene expression group's cell-types-analysis package" - or something similar to your taste.